machine-learning/tutorials/SentimentAnalysis/Program.cs

```diff
@@ -191,9 +193,11 @@ public static void UseLoadedModelWithBatchItems(MLContext mlContext)
 
             // <SnippetLoadModel>
             ITransformer loadedModel;
+            DataViewSchema dataViewSchema;
```

Comment:
> DataViewSchema dataViewSchema; [](start = 11, length = 31)

maybe add comment: a variable to store the schema of the model, generated during loading.